src/lib/crypto/utils.ts

```diff
@@ -9,6 +9,29 @@ export { bech32 } from 'bech32'
 
 export { PrivateKey, PublicKey, PublicKeyHash, KeyedSignature, RecoverableSignature, Signature } from "./types"
 
+export const BigMath = {
+    abs(x: bigint): bigint {
+        return x < 0n ? -x : x
+    },
+    sign(x: bigint): bigint {
+        if (x === 0n) return 0n
+        return x < 0n ? -1n : 1n
+    },
+    pow(base: bigint, exponent: bigint): bigint {
+        return base ** exponent
+    },
+    min(value: bigint, ...values: bigint[]): bigint {
+        for (const v of values)
+            if (v < value) value = v
+                return value
+    },
+    max(value: bigint, ...values: bigint[]): bigint {
+        for (const v of values)
+            if (v > value) value = v
+                return value
+    },
+}
+
```

Comment:
Are we using this methods in our codebase?

```suggestion
```

Reply:
Please remove any unnecessary or unused code to help prevent potential errors and reduce untested logic. Following the [YAGNI principle](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) ensures we only include what we truly need right now